inc/server/class-form-server.php

```diff
@@ -949,6 +1024,16 @@ public function pull_fields_options_for_form( $form_data ) {
 		return $form_data;
 	}
 
+	/**
+	 * Verify the given confirmation session.
+	 *
+	 * @param string $session The session id.
+	 * @return bool
+	 */
+	public function verify_confirmation_session( $session ) {
+		// TODO: Add verification for Stripe session when adding the stripe field.
+		return 'test_123' === $session; // Test ONLY.
```

Comment:
Here will add the Stripe verification process when we check if the given session is still valid.